src/test/java/nl/praegus/fitnesse/symbols/MavenProjectVersions/MavenProjectVersionsSymbolTest.java

```diff
@@ -0,0 +1,48 @@
+package nl.praegus.fitnesse.symbols.MavenProjectVersions;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MavenProjectVersionsSymbolTest {
+    @Test
+    public void compare_If_Current_Version_Is_Outdated() {
+        MavenProjectVersionsSymbol MavenVersionSymbol = new MavenProjectVersionsSymbol();
+        String currentVersion = "1.1.0";
+        String latestVersion = "1.2.0";
+
+        String receivedValue = MavenVersionSymbol.getStatus(currentVersion, latestVersion);
+
+        assertThat(receivedValue).isEqualTo("Outdated");
```

Comment:
De logica van versionNumber testen we nu in speciale tests daarvoor (wat ik eerder al suggereerde). Wat je nu hier doet redundant. Ik denk 1 test meer dan genoeg is om de 3 regels in mavenprojectversionssymbol.getstatus te testen.

Reply:
Redundant testen weggehaald